src/App/Components/Common/NavProfile/Profile.js

```diff
@@ -123,12 +154,17 @@ export class Profile extends Component {
               <a className="dropdown-item dropdown-item-custom" href="/profile">
                 Profile
               </a>
-              <a className="dropdown-item dropdown-item-custom" href="/">
+              <a
+                className="dropdown-item dropdown-item-custom"
```

Comment:
@truestbyheart A button would be better here. see this: https://bitsofco.de/anchors-vs-buttons/